cmd/svcinit/main.go

```diff
@@ -97,16 +98,30 @@ func main() {
 	serviceSpecs, err := augmentServiceSpecs(unversionedSpecs, ports)
 	must(err)
 
-	/*if *allowSvcctl {
-		addr := net.Listen(network, address)
-	}*/
-
 	ctx, cancelFunc := context.WithCancel(context.Background())
 	defer cancelFunc()
 
 	r, err := runner.New(ctx, serviceSpecs)
 	must(err)
 
+	servicesErrCh := make(chan error, len(serviceSpecs))
+
+	if _, ok := os.LookupEnv("SVCINIT_TEST_ALLOW_SVCCTL"); ok {
```

Comment:
any downside to always enabling it? Seems like it will be less confusing to always have it on.
Also, I was thinking of using a well-known port in non-test mode, that way when you `bazel run` a service group, we can CLI to lookup ports for the managed services, wdyt?